unit_tests/storage/storage_cuda/test_data_view.cu

```diff
@@ -24,7 +24,11 @@ template <typename View>
 __global__ void mul2(View s) {
     bool correct_dims = (s.template total_length<0>() == c_x) && (s.template total_length<1>() == c_y) &&
                         (s.template total_length<2>() == c_z);
+#ifdef __HIPCC__
+    bool correct_size = (s.padded_total_length() == 16 * c_y * c_z);
```

Comment:
Can we instead query the alignment from  the storage_info, would make the test more readable.